<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EducationalDetails</class>
 <widget class="QDialog" name="EducationalDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="FscCollege">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color :transparent;
	font: 11pt &quot;Georgia&quot;;
color :black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 11pt &quot;Georgia&quot;;
}</string>
   </property>
   <property name="text">
    <string>Matric Marks :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Fscmarks">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color :transparent;
	font: 11pt &quot;Georgia&quot;;
color :black;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MatricSchool">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color :transparent;
	font: 11pt &quot;Georgia&quot;;
color :black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButten_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 11pt &quot;Georgia&quot;;
}</string>
   </property>
   <property name="text">
    <string>Matric School :</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 11pt &quot;Georgia&quot;;
}</string>
   </property>
   <property name="text">
    <string>Fsc Marks :</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 11pt &quot;Georgia&quot;;
}</string>
   </property>
   <property name="text">
    <string>Fsc College :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButten_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;
font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string>&lt; Logout</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MarticMarks">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color :transparent;
	font: 11pt &quot;Georgia&quot;;
color :black;
}</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background-color: #417BFD;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BackButten_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Personal Details</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButten_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Educational details</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButten_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Admission Fees</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButten_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: #417BFD;
color: white;
border: 2px solid white;
border-radius: 10px;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;

}
QPushButton:hover
{
background-color: white;
color: #417BFD;
border: 3px solid #417BFD;
	font: 11pt &quot;Georgia&quot;;

font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>&lt;Back</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
